client/src/ui/FilteredResult.tsx

```diff
@@ -128,7 +128,7 @@ export default class FilteredResult extends Component<Props,State> {
               <strong>{this.updateSortByTitle()}</strong>
             </p>
             <p className="result-card-sort-by-value">
-              {this.updateSortBy()}
+              {isNaN(Number(this.updateSortBy())) ? this.updateSortBy() : Number(this.updateSortBy()).toFixed(1)}
```

Comment:
Two thoughts:
- I think we should change the name of `updateSortBy` to something more like `getSortNumber`
- I think the logic for rounding should be an optional parameter to this function
  - E.g. `getSortNumber(1)` for getting number rounded to 1 decimal; `getSortNumber()` for no rounding